docs/partitioning.md

```diff
@@ -28,9 +31,9 @@ work regardless of whether partition is an actual partition context or `None`, m
 no partition is specified.
 
 When dealing with state, for example, developers may always call
-`Stream.get_context_state(context)` even if `context` is not set.
-The method will automatically return the state that is appropriate, either for the partition
-or for the stream.
+[`Stream.get_context_state(context)`](singer_sdk.Stream.get_context_state) even if
```

Comment:
I had no idea this type of link syntax to module ref docs would work. Very cool!

Reply:
@edgarrmondragon is this provided by a Sphinx plugin? If so, which one?

Reply:
@WillDaSilva this is [MyST](https://myst-parser.readthedocs.io/en/latest/) iirc